build/items.json

```diff
@@ -2950,7 +2975,7 @@
   },
   "dagon": {
     "hint": [
-      "Active: Energy Burst Emits a powerful burst of magical damage upon a targeted enemy unit. Upgradable.Damage: 400,500,600,700,800Range: 700,750,800,850,900Mana Cost: 120,140,160,180,200"
+      "Active: Energy Burst Emits a powerful burst of magical damage upon a targeted enemy unit. Upgradable.Damage: undefinedRange: undefinedMana Cost: undefined"
```

Comment:
Those undefined are fixed by this change : https://github.com/odota/dotaconstants/pull/118/files/1117233fd9577833af3ddab13001813480758c33#diff-0422506b30bda77d61e9f550663e1a1ccb0eb94be4104f27c79d619b1dda91acL1103-L1106